samples/settings/boards/esp32.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_WIFI=y
+CONFIG_WIFI_ESP32=y
+CONFIG_HEAP_MEM_POOL_SIZE=131072
+
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_DHCPV4=y
+
+CONFIG_NET_CONFIG_LOG_LEVEL_DBG=y
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+
+CONFIG_MBEDTLS_ENTROPY_ENABLED=y
+CONFIG_MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED=y
+CONFIG_MBEDTLS_ECP_ALL_ENABLED=y
+
+
+CONFIG_ESP32_WIFI_STA_AUTO=y
+CONFIG_GOLIOTH_SAMPLE_WIFI=n
```

Comment:
GOLIOTH_SAMPLE_WIFI is disabled by default, so no reason to explicitly deselect it.